src/e2e-tests/features/autelis.feature

```diff
@@ -0,0 +1,96 @@
+@autelis
+Feature: RoboDomo Autelis page
+
```

Comment:
Please add a description to this feature page. Usually the scope of the scenarios